### PR TITLE
Fixed Quota error by multiplying services per account and region.

### PR DESCRIPTION
**Pull request by zagronitay**
https://docs.aws.amazon.com/AWSEC2/latest/APIReference/throttling.html

https://github.com/cloudquery/cloudquery/issues/94

### CODE REVIEW

File: client/client.go [hunk 1]
```diff
@@ -95,15 +97,36 @@ type Services struct {
...
 }
 
+type ServicesAccountRegionMap map[string]map[string]*Services
+
+// ServicesManager will hold the entire map of (account X region) services
+type ServicesManager struct {
+	services ServicesAccountRegionMap
+}
+
+func (s *ServicesManager) ServiceByAccountAndRegion(accountId string, region string) *Services {
```
Comment (roneli) [inline client/client.go:107]: Services*ByAccountAndRegion

File: client/client.go [hunk 2]
```diff
@@ -67,6 +67,8 @@ var allRegions = []string{
 	"sa-east-1",
 }
 
+var defaultRegion = "us-east-1"
```
Comment (roneli) [inline client/client.go:70]: const

**Review by roneli — APPROVED** [review]
Looks good, minor suggestions (grammar + const)

**Review by yevgenypats — CHANGES_REQUESTED** [review]
I've tested it and Im not sure it solved the problem. The AWS client should be goroutine safe per documentation also the retrier is automatically copied like all other options on each call. For example see this : https://github.com/aws/aws-sdk-go-v2/blob/main/service/autoscaling/api_client.go#L121 

I suspect this is something else like incorrect retrier configuration in the first place. I've a few ideas but let's discuss this live tomorrow

@roneli @zagronitay

**Review by yevgenypats — APPROVED** [review]

**Pull request merged**